tensorflow/core/kernels/ragged_tensor_to_tensor_op.cc

```diff
@@ -381,13 +399,13 @@ class RaggedTensorToTensorBaseOp : public OpKernel {
     OP_REQUIRES_OK(context, GetFirstDimensionSize(context, &first_dimension));
     vector<INDEX_TYPE> output_size;
     OP_REQUIRES_OK(context,
-                   CalculateOutputSize(first_dimension, context, &output_size));
+                   CalculateOutputSize(first_dimension, context, &output_size)); 
```

Comment:
```suggestion
                   CalculateOutputSize(first_dimension, context, &output_size));
```

No trailing whitespace, please